SYCL/SubGroup/info.cpp

```diff
@@ -24,7 +24,7 @@ int main() {
 
   /* Basic sub-group functionality is supported as part of cl_khr_subgroups
    * extension or as core OpenCL 2.1 feature. */
-  if (!core_sg_supported(Device)) {
+  if (!core_sg_supported(Device) || !Device.has(aspect::fp64)) {
```

Comment:
```suggestion
  if (!core_sg_supported(Device)) {
```

https://github.com/intel/llvm-test-suite/pull/1242 is better than this one.